pkg/util/iptables/iptables.go

```diff
@@ -150,6 +150,7 @@ func New(exec utilexec.Interface, dbus utildbus.Interface, protocol Protocol) In
 		hasCheck: getIPTablesHasCheckCommand(vstring),
 		waitFlag: getIPTablesWaitFlag(vstring),
 	}
+	// TODO should we move this to a separate Start() or Run() function?
 	runner.connectToFirewallD()
```

Comment:
Yeah, this is pretty ugly to do in a util library, but there are a couple callers to fix up.